expression/aggregation/aggregation.go

```diff
@@ -49,6 +50,9 @@ type Aggregation interface {
 	// GetResult will be called when all data have been processed.
 	GetResult(ctx *AggEvaluateContext) types.Datum
 
+	// AppendResultToChunk will be called when all data have been processed.
+	AppendResultToChunk(chunk *chunk.Chunk, colIdx int, ctx *AggEvaluateContext)
```

Comment:
You can now remove this change, we'll refactor all the aggregation functions and make them support Chunk in the near future.